plugins/woocommerce-blocks/assets/js/blocks/product-filters/edit.tsx

```diff
@@ -121,8 +137,179 @@ export const Edit = ( {}: BlockEditProps< ProductFiltersBlockAttributes > ) => {
 		return <Spinner />;
 	}
 
+	const templatePartEditUri = getSetting(
+		'templatePartProductFiltersOverlayEditUri',
+		''
+	) as string;
```

Comment:
You can avoid type-casting by providing the type of returned value:
```suggestion
	const templatePartEditUri = getSetting< string >(
		'templatePartProductFiltersOverlayEditUri',
		''
	);
```

Reply:
Or you can also use `getSettingWithCoercion` for extra safety, but IMO, in this case, `getSetting` is enough.